core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -378,8 +379,13 @@ public void removeBlock(long sessionId, long blockId)
   public void removeBlock(long sessionId, long blockId, BlockStoreLocation location)
```

Comment:
Is this only called by the async block remover? Are there any other callers which may be affected by the new locking behavior?